People/app/src/main/java/com/example/android/people/data/NotificationHelper.kt

```diff
@@ -70,55 +64,6 @@ class NotificationHelper(private val context: Context) {
                 }
             )
         }
-        updateShortcuts(null)
-    }
-
-    @WorkerThread
-    fun updateShortcuts(importantContact: Contact?) {
-        var shortcuts = Contact.CONTACTS.map { contact ->
-            val icon = IconCompat.createWithAdaptiveBitmap(
```

Comment:
After removing this, notification UI changed on Android 13 to:

<img src="https://user-images.githubusercontent.com/10363352/231695728-32e5b52a-49b4-4c02-9473-af9513aef7d2.png" width="25%" height="25%">